biogeochem/EDCohortDynamicsMod.F90

```diff
@@ -150,7 +204,7 @@ subroutine create_cohort(currentSite, patchptr, pft, nn, hite, dbh, bleaf, bfine
     ! Assign canopy extent and depth
     call carea_allom(new_cohort%dbh,new_cohort%n,spread,new_cohort%pft,new_cohort%c_area)
 
-    new_cohort%treelai = tree_lai(new_cohort%bl, new_cohort%pft, new_cohort%c_area,    &
+    new_cohort%treelai = tree_lai(bleaf, new_cohort%pft, new_cohort%c_area,    &
```

Comment:
Just curious why the calculation of LAI here is slightly different than how it is done in biogeochem/EDCanopyStructureMod.F90 where here we have bleaf (leaf biomass) and above leaf_c which is also leaf biomass?  Just trying to understand the steps....

Reply:
@serbinsh , the stuff that mostly happens in EDCanopyStructureMod, is the tallying of LAI from each cohort, and assembling those LAI's into canopy, leaf and PFT bins within each patch.  But that part of the code shouldn't be calculating the individual plant's LAI any differently.  Could you point to the line in question so that I can double check?